allauth/account/views.py

```diff
@@ -290,6 +290,13 @@ def get_context_data(self, **kwargs):
         )
         return ret
 
+    def get_initial(self):
+        if self.request.GET.get("email", None):
+            initial = super().get_initial()
+            initial["email"] = self.request.GET.get("email")
```

Comment:
In case of `app_settings.SIGNUP_EMAIL_ENTER_TWICE` there is also an `email2` field. That would need to be prefilled as well.

Reply:
Should we restrict if user is prefilling only one field, ex: only email or email2. Or is it fine to prefill anyone or both?

Reply:
It is fine to prefill both.